### PR TITLE
Password reset changes

### PR DESCRIPTION
**Pull request by suehighmoor**
Updated:
How to change your email address in the Portal - removed detailed steps for changing login email address, added detail and screenshots for changing password reset email address
How to reset your password or memorable word - added section for resetting password, removed any refs to admins resetting user passwords
Portal user access - updated section about resetting passwords
Best practice customer guide - added section for setting password reset email

### CODE REVIEW

**Review by SteveHallUKCLoud — APPROVED** [review]
LGTM

**Comment by suehighmoor** [comment]
> I can't tell if articles/portal/images/ptl-update-user-password.png has been updated or removed? Presumably it is no longer needed.

I had just updated the image as admin can still change another user's memorable word. However, I realise now that this won't reflect how the UI will look when the ability to change the password is removed, as presumably the password fields will no longer be there. So I've removed the image for now.

File: articles/portal/ptl-how-change-email-address.md [hunk 1]
```diff
@@ -1,65 +1,57 @@
...
 
-## Changing your login email address
+- Your password reset email address is the one that password reset links are sent to if you forget your password and request to reset it. By default this the same as your login email address. However, if you prefer you can change this to be a different email address as described in this article.
 
-1. [*Log in to the UKCloud Portal*](ptl-gs.md#logging-in-to-the-ukcloud-portal) using your old email address.
+## Setting your password reset email address
 
-2. [*Use My Calls to raise a Service Request*](ptl-how-use-my-calls.md).
+To ensure that can still access your Portal account if you forget your password, it's important that you set your password reset email address to one that you can readily access.
```
[the remark this answers is not in the document]
Comment (suehighmoor) [reply articles/portal/ptl-how-change-email-address.md]: Fixed

**Comment by suehighmoor** [comment]
@bensaunders, back to you if you don't mind :)

**Review by bensaunders — APPROVED** [review]

**Pull request merged**